resources/variants/cartesio_0.25.inst.cfg

```diff
@@ -0,0 +1,15 @@
+[general]
+name = 0.25 mm
+version = 2
+definition = cartesio
+
+[metadata]
+author = Cartesio
+type = variant
+
+[values]
+machine_nozzle_size = 0.25
+machine_nozzle_tip_outer_diameter = 1.05
+speed_wall = =round(speed_print / 1.25, 1)
```

Comment:
I haven't really crystallized these settings yet, but wanted to be ready for the future :)